backends/dialogs/win32/win32-dialogs.cpp

```diff
@@ -134,7 +134,7 @@ Common::DialogManager::DialogResult Win32DialogManager::showFileBrowser(const ch
 		hr = dialog->SetTitle(str);
 		delete[] str;
 
-		str = Win32::ansiToUnicode(_("Choose"), Win32::getCurrentCharset());
+		str = Win32::ansiToUnicode(Common::convertFromU32String(_("Choose")).c_str(), Win32::getCurrentCharset());
```

Comment:
So, you're converting from Unicode, to Unicode, and then back to Unicode? Please, rethink this.

Reply:
this was my bad! rethought about this, new code pushed and tested. thanks!